boards/arm/secureiot1702/doc/board.rst

```diff
@@ -0,0 +1,18 @@
+.. _secureiot1702:
+
+Microchip SecureIoT1702
+#######################
+
+Board configuration for Microchip CEC1702 demo board. This is
```

Comment:
Thanks for getting the documentation started. We've got a recommended template for board documentation at doc/templates/board.tmpl (in your cloned copy of the zephyr tree).

Reply:
Ok, will take it from there. I also have pending patches for IRQ support, and QMSPI host driver for CEC. As well as generic SPI flash (auto-detects way to access chip using SFDP) driver. I will possibly also work on gpio, pinmux, clock drivers.